### ISSUE TITLE
Type ProjectsGroupsModel.Project missing targetLanguages property

### ISSUE THREAD

**Issue by Alain00** [crowdin/crowdin-api-client-js#79]
#### Example
```js
projectsGroupsApi.getProject(417404).then(res=>{
    console.log(res.data.targetLanguages) //Property 'targetLanguages' does not exist on type 'Project | ProjectSettings'
})
```

#### Log
```
Array [
    Object {
      "androidCode": "es-rES",
      "dialectOf": null,
      "editorCode": "es",
      "id": "es-ES",
      "locale": "es-ES",
      "name": "Spanish",
      "osxCode": "es.lproj",
      "osxLocale": "es",
      "pluralCategoryNames": Array [
        "one",
        "other",
      ],
      "pluralExamples": Array [
        "1",
        "0, 2-999; 1.2, 2.07...",
      ],
      "pluralRules": "(n != 1)",
      "textDirection": "ltr",
      "threeLettersCode": "spa",
      "twoLettersCode": "es",
    },
    Object {
      "androidCode": "en-rUS",
      "dialectOf": null,
      "editorCode": "en",
      "id": "en",
      "locale": "en-US",
      "name": "English",
      "osxCode": "en.lproj",
      "osxLocale": "en",
      "pluralCategoryNames": Array [
        "one",
        "other",
      ],
      "pluralExamples": Array [
        "1",
        "0, 2-999; 1.2, 2.07...",
      ],
      "pluralRules": "(n != 1)",
      "textDirection": "ltr",
      "threeLettersCode": "eng",
      "twoLettersCode": "en",
    },
  ]
```

**Comment by yevheniyJ** [comment]
@Alain00 Hi there! Thanks for highlighting this issue. Looks like this field was added recently. Let me double check it and will return to you with an update^^

**Comment by yevheniyJ** [comment]
@Alain00 Fixed and new version available 1.8.15
